railties/test/application/rake/single_resource_test.rb

```diff
@@ -0,0 +1,30 @@
+require "isolation/abstract_unit"
+require "active_support/core_ext/string/strip"
+class RakeTest < ActiveSupport::TestCase
+  include ActiveSupport::Testing::Isolation
+  def setup
+    build_app
+  end
+
+  def teardown
+    teardown_app
+  end
+  def test_26606
```

Comment:
Can you write a better test name for this? Maybe `test_singular_resource_output_in_rake_routes`